backend-config/types.go

```diff
@@ -40,6 +40,7 @@ type SourceDefinitionT struct {
 	ID       string
 	Name     string
 	Category string
+	Type     string // // Indicates whether source is one of {cloud, web, flutter, android, ios, warehouse, cordova, amp, reactnative, unity}. This field is not present in sources table
```

Comment:
Is this a new field that control plane is going to provide?

Reply:
Yes

Reply:
fyi, in the future we'll be switching to the v2 config api, now [in progress](https://github.com/rudderlabs/rudder-cp-sdk/blob/initial-setup/model/model.go#L74-L83)